src/core/rate_limiter.rs

```diff
@@ -1,5 +1,7 @@
 use pyo3::prelude::*;
 use std::sync::Mutex;
+use std::time::{SystemTime};
+// use log::{debug};
```

Comment:
@brettlangdon How should I include the logging crate? I am following this doc and it seems like I am missing something: https://crates.io/crates/log (can I install rust crates via riot?).

Reply:
we talked offline, there is the pyo3-log crate which helps log in Python from rust that should be used.